src/contracts/Model/Suggestion/SuggestionCollection.php

```diff
@@ -9,7 +9,7 @@
 namespace Ibexa\Contracts\Search\Model\Suggestion;
 
 use Ibexa\Contracts\Core\Collection\MutableArrayList;
-use Ibexa\Contracts\Core\Exception\InvalidArgumentException;
```

Comment:
We should type against contracts if that is possible, any reason behind those changes?